Project Force Deletion (#1107)
spectrocloud/librarium: content/docs/08-user-management/1.8-project-association.md

---
title: "Project Association"
metaTitle: "Associate a User or Team with a Project"
metaDescription: "Associate a User or Team with a Project"
hideToC: false
fullWidth: false
---

# Overview

> Associating a user or team with a specific project creates a clear distinction between who is allowed in the project and their access control permissions. By grouping resources together, you ensure that only the designated members have access to and control over the project resources, preventing others from accidentally or intentionally modifying them. This improves resource accountability, reduces confusion and conflicts, and helps maintain the integrity and security of the project.

Reviewer: ```suggestion
Associating a user or team with a specific project creates a clear distinction between who is allowed in the project and their access control permissions. By grouping resources together, you ensure that only designated members have access to and control over the project resources, preventing others from accidentally or intentionally modifying them. This improves resource accountability, reduces confusion and conflicts, and helps maintain the integrity and security of the project.
```

> The user's permissions are determined by the combination of their tenant and project roles, as well as any roles inherited from their team association. If a user is a *Tenant Admin*, they have project admin permissions in all projects. A user with the *Project Viewer* role at the tenant level will have *View* permissions for all projects. However, if a user or team has the *Project Viewer* role assigned specifically to a project, they will only have viewing access to that project. The extent of the user's permissions, either at the tenant or project level, determines the number of projects they can access.

Reviewer: ```suggestion
User permissions are determined by the combination of their tenant and project roles, as well as any roles inherited from their team association. If a user is a *Tenant Admin*, they have admin permissions in all projects. A user with the *Project Viewer* role at the tenant level has *View* permissions for all projects. However, if a user or team has the *Project Viewer* role assigned to a specific project, they only have view access to that project. The extent of user permissions, either at the tenant or project level, determines the number of projects they can access.
```

Reviewer: I try to avoid using possessive unless it's really needed.

# Associate a User or Team

> To associate a user or a team with a project, use the following steps.

Reviewer: ```suggestion
To associate a user or team with a project, use the following steps.
```

# Prerequisites

> * Tenant Admin access

Reviewer: ```suggestion
* Tenant Admin access.
```

* An available project. Check out the [Create a Project](/projects#createaproject) guide to learn how to create a project.

* A user or a team.

# Enablement

> 1. Log in to [Palette](https://console.spectrocloud.com)

Reviewer: ```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

2. Select the **Tenant Admin** scope.

3. Navigate to the left **Main Menu** and select **Users & Teams**.

4. Select the tab of the resource you want to associate with a project, either a user or a team.

5. Click on the row of the user or team to display its overview page.

6. Select **Project Roles**.

7. A **drop-down Menu** containing all the available projects in the tenant is available. Select the project you want to associate with the project role.

8.  Next, assign permissions to the project role. To learn more about each permission and available roles, check out the [Palette RBAC](/user-management/palette-rbac) documentation.

> Click on **Confirm** to create the project role and complete the project association process.

Reviewer: ```suggestion
Click **Confirm** to create the project role and complete the project association process.
```
